cli/tests/integrations/test_marathon_pod.py

```diff
@@ -1,26 +1,17 @@
 import contextlib
 import json
-import os
 import re
 
 from dcos import util
 
 import pytest
 
-from ..common import assert_same_elements
-from .common import assert_command, exec_command, file_bytes, file_json
-
-FILE_PATH_BASE = 'tests/data/marathon/pods'
-
-GOOD_POD_ID = 'good-pod'
-GOOD_POD_FILE_PATH = os.path.join(FILE_PATH_BASE, 'good.json')
-UPDATED_GOOD_POD_FILE_PATH = os.path.join(FILE_PATH_BASE, 'updated_good.json')
-
-DOUBLE_POD_ID = 'double-pod'
-DOUBLE_POD_FILE_PATH = os.path.join(FILE_PATH_BASE, 'double.json')
-
-TRIPLE_POD_ID = 'winston'
-TRIPLE_POD_FILE_PATH = os.path.join(FILE_PATH_BASE, 'doubleplusgood.json')
+from ..fixtures.marathon import (DOUBLE_POD_FILE_PATH, DOUBLE_POD_ID,
```

Comment:
Well, that's kind of why I moved those constants to `fixtures.marathon`, because that module only defines test data. Should I just do a wildcard import of `fixtures.marathon`? Or do you think I should make a `constants.marathon` module?